_data/es.yml

```diff
@@ -274,26 +274,26 @@ faq:
+    <p>La DAO Bisq (organización autónoma descentralizada) es un mecanismo de gobernanza descentralizada para el software Bisq. Toma el lugar de una compañía. ¿Cuán bueno puede ser un software descentralizado si está controlado por una única entidad, como una compañía? </p> <p>El software descentralizado no es bueno sin gobernanza descentralizada.</p> <p>Esta es la razón por la que Bisq no es una compañía o una entidad legal de ningún tipo. Está organizada como una DAO. La DAO se encarga de dotar de fondos el software y crear la estrategia para que la red Bisq prospere, y al mismo tiempo, endurecerse contra ataques en la infraestructura, reforzando su liderazgo y operativa. </p> <p>En unas palabras, la DAO Bisq permite a Bisq ser aún más resistente a la censura, un <a href='https://docs.bisq.network/intro.html' target='_blank'>principio central del proyecto</a> desde el principio.</p> <p>Lea más acerca de la DAO Bisq en <a href='https://docs.bisq.network/user-dao-intro.html' target='_blank'>este documento introductorio</a>.</p>
   hBisqDaoHow: "¿Cómo funciona la DAO Bisq?"
   pBisqDaoHow: >
-    <p>Los propósitos primarios de la DAO Bisq son financiar el proyecto y determinar la estrageia.</p> <ul>
+    <p>Los propósitos primarios de la DAO Bisq son financiar el proyecto y determinar la estrategia.</p> <ul>
         <li>Las comisiones de intercambio se distribuyen directamente desde los comerciantes a los contribuyentes con un token (bitcoin coloreado) llamado BSQ.</li>
         <li>La estrategia se determina colectivamente a través de la votación hecha en el software Bisq y registrada en la red Bitcoin.</li>
     </ul> <p>Aprenda más acerca de cómo funciona en realidad con <a href='https://www.youtube.com/playlist?list=PLFH5SztL5cYPAXWFz-IMB4dBZ0MEZEG_e' target='_blank'>esta serie de videos cortos de 3-5 minutos</a>.</p>
-  hNeedToKnow: "¿Como comerciante, qué necesito saber acerca de la DAO?"
+  hNeedToKnow: "¿Cómo comerciante, qué necesito saber acerca de la DAO?"
```

Comment:
The original, without accent, was fine.

"Como" en este caso, aunque sea pregunta, no tiene valor interrogativo, con lo que va sin acento (preposición). https://www.fundeu.es/recomendacion/como-con-sin-tilde-diferencia/